docs/walkthrough/working-with-elasticsearch.md

```diff
@@ -1,44 +1,23 @@
 <!-- markdownlint-disable -->
 
-## About ElasticSearch
-
 The [Quilt web catalog
 search](../walkthrough/working-with-the-catalog#search) is powered
-by ElasticSearch. To write specialized queries against your data
+by Elasticsearch. To write specialized queries against your data
 stored in Amazon S3 buckets you may wish to connect directly to
-your Quilt ElasticSearch cluster.
-
-> Note that Quilt is currently pinned to ElasticSearch 6.7
+your Quilt Elasticsearch cluster.
 
-Each Amazon S3 bucket connected to Quilt has two ElasticSearch indexes:
-1. `<s3-bucket>`: For S3 object documents
-2. `<s3-bucket>_packages`: For Quilt data package documents
+> Note that Quilt is currently pinned to Elasticsearch 6.7
 
-Provided you have IAM permissions, you can write queries against
-the indexes which will search across all your Amazon S3 buckets
-connected to Quilt.
+Each Amazon S3 bucket connected to Quilt has two Elasticsearch indexes
+with the following aliases:
+1. `YOUR_BUCKET_NAME`: Contains one document per object in the bucket.
+2. `YOUR_BUCKET_NAME_packages`: Contains one document per package revision in the bucket.
 
-## Connecting to your indexes
+## Querying Elasticsearch with Python
 
-Before writing your specialized queries, you will need to ensure
-that you have authenticated and have access to AWS resources. The
-best way to do this is to [configure your AWS CLI
-credentials](https://docs.quiltdata.com/more/faq#do-i-have-to-login-via-quilt3-to-use-the-quilt-apis-how-do-i-push-to-quilt-from-a-headless-environme).
-
-<!--pytest.mark.skip-->
-```bash
-% export AWS_PROFILE=<your-aws-profile>
-```
-
-You can then connect directly to the ElasticSearch cluster to write
-custom queries. This is faster than writing queries to multiple S3
-buckets due to how registries are laid out in Amazon S3. 
-
-### Example
-
-Below is an example using Python to search the Quilt data package 
-documents index (`*_packages`) across all S3 buckets (`*`), 
-returning the top `1000` results.
+You can use the [`Elasticsearch`
+API](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/) to
+query your cluster.
```

Comment:
This is a PyPI module, should be lower case and here is pinned to the highest version less than or
equal to 6.7 for compatibility.
```suggestion
You can use [`elasticsearch 6.3.1`](https://elasticsearch-py.readthedocs.io/en/6.3.1/) as follows.
```